databricks/koalas/tests/test_groupby.py

```diff
@@ -344,80 +345,96 @@ def test_bfill(self):
+                           pdf.groupby(['a', 'b']).apply(lambda x: x * x).sort_index())
+            self.assert_eq(kdf.groupby(['b'])['a'].apply(lambda x: x).sort_index(),
+                           pdf.groupby(['b'])['a'].apply(lambda x: x).sort_index())
+
+            # Data is intentionally big to test when schema inference is on.
+            pdf = pd.DataFrame({'a': [1, 2, 3, 4, 5, 6] * 300,
+                                'b': [1, 1, 2, 3, 5, 8] * 300,
+                                'c': [1, 4, 9, 16, 25, 36] * 300}, columns=['a', 'b', 'c'])
+            kdf = koalas.DataFrame(pdf)
+            self.assert_eq(kdf.groupby("b").apply(lambda x: x + 1).sort_index(),
+                           pdf.groupby("b").apply(lambda x: x + 1).sort_index())
+            self.assert_eq(kdf.groupby(['a', 'b']).apply(lambda x: x * x).sort_index(),
+                           pdf.groupby(['a', 'b']).apply(lambda x: x * x).sort_index())
+            self.assert_eq(kdf.groupby(['b'])['a'].apply(lambda x: x).sort_index(),
+                           pdf.groupby(['b'])['a'].apply(lambda x: x).sort_index())
+            with self.assertRaisesRegex(TypeError, "<class 'int'> object is not callable"):
+                kdf.groupby("b").apply(1)
+        finally:
+            reset_option('compute.shortcut_limit')
 
     def test_apply_with_new_dataframe(self):
-        # Less than 1000 records will execute a shortcut by using collected pandas dataframe
-        # directly.
-        pdf = pd.DataFrame({
-            "timestamp": [0.0, 0.5, 1.0, 0.0, 0.5],
-            "car_id": ['A', 'A', 'A', 'B', 'B']
-        })
-        kdf = koalas.DataFrame(pdf)
-
-        self.assert_eq(
-            kdf.groupby('car_id').apply(lambda _: pd.DataFrame({"column": [0.0]})).sort_index(),
-            pdf.groupby('car_id').apply(lambda _: pd.DataFrame({"column": [0.0]})).sort_index())
-
-        # 1000+ records will only infer the schema.
-        pdf = pd.DataFrame({
-            "timestamp": [0.0, 0.5, 1.0, 0.0, 0.5],
-            "car_id": ['A', 'A', 'A', 'B', 'B']
-        })
-        kdf = koalas.DataFrame(pdf)
-
-        self.assert_eq(
-            kdf.groupby('car_id').apply(lambda _: pd.DataFrame({"column": [0.0]})).sort_index(),
-            pdf.groupby('car_id').apply(lambda _: pd.DataFrame({"column": [0.0]})).sort_index())
+        set_option('compute.shortcut_limit', 1000)
+
+        try:
+            pdf = pd.DataFrame({
+                "timestamp": [0.0, 0.5, 1.0, 0.0, 0.5],
+                "car_id": ['A', 'A', 'A', 'B', 'B']
+            })
+            kdf = koalas.DataFrame(pdf)
+
+            self.assert_eq(
+                kdf.groupby('car_id').apply(lambda _: pd.DataFrame({"column": [0.0]})).sort_index(),
+                pdf.groupby('car_id').apply(lambda _: pd.DataFrame({"column": [0.0]})).sort_index())
+
+            # 1000+ records will only infer the schema.
+            pdf = pd.DataFrame({
+                "timestamp": [0.0, 0.5, 1.0, 0.0, 0.5],
```

Comment:
There seems a mistake here. Can you `"timestamp": [0.0, 0.5, 1.0, 0.0, 0.5],` -> `"timestamp": [0.0, 0.5, 1.0, 0.0, 0.5] * 300,` to make it 1000+ records?

Reply:
@HyukjinKwon Thanks to comment. i've fixed them :)